target/scripts/startup/setup-stack.sh

```diff
@@ -1183,6 +1183,39 @@ EOF
   supervisorctl update
 }
 
+function _setup_getmail
+{
+  _log 'trace' 'Preparing Getmail configuration'
+
+  local CONFIGURATION GETMAILRC
+
+  CONFIGURATION='/tmp/docker-mailserver/getmail-*.cf'
+  GETMAILRC='/etc/getmailrc.d'
+
+  if [[ ! -d ${GETMAILRC} ]]
+  then
+    mkdir "${GETMAILRC}"
+  fi
+
+  for FILE in ${CONFIGURATION}; do
+    if [[ -f ${FILE} ]]
+    then
+      USER=$(echo "${FILE}" | cut -d'-' -f 3| cut -d'.' -f1)
+      sed "s/PLACEHOLDER/${USER}/g" /etc/getmailrc_general > "${GETMAILRC}/getmailrc-${USER}.tmp"
+      cat "${GETMAILRC}/getmailrc-${USER}.tmp" "${FILE}" > "${GETMAILRC}/getmailrc-${USER}"
+      rm "${GETMAILRC}/getmailrc-${USER}.tmp"
+      GETMAIL_POLL_MINS=$((GETMAIL_POLL/60))
+      cat >"/etc/cron.d/getmail-${USER}" << EOF
+*/${GETMAIL_POLL_MINS} * * * * root /usr/bin/getmail --getmaildir /var/lib/getmail --rcfile "${GETMAILRC}/getmailrc-${USER}"
+EOF
+    else
+      cat /etc/getmailrc_general > "${GETMAILRC}/getmailrc"
+    fi
+  done
+  chmod -R 600 "${GETMAILRC}"
+  chown -R root:root "${GETMAILRC}"
+}
```

Comment:
Same concern here with the `USER` as mentioned in an earlier review comment.

This answers the concern about `PLACEHOLDER` and default configs with the `sed` + `cat` lines.

Creating a separate cron job per user config seems like a :-1: from me. I think it'd be better to have a single cron job that can loop the command for each `--rcfile` arg? (_like you had in your `germail-wrapper.sh` script?_)

Is there a need for the `chmod` and `chown`? The `chmod` is fine, but `chown` should already be `root:root`?

Reply:
Just to clarify, is `GETMAIL_POLL_MINS` the reason for citing a minimum of 60 seconds? That seems to be the case and perhaps related to your switch to using cron.

I would advise to instead, change the units from seconds to minutes. Simplifies the whole thing.

Reply:
Yep, will do, I went with seconds to keep it consistent with fetchmail.